CHANGELOG-PENDING.md

```diff
@@ -12,6 +12,7 @@ Month, DD, YYYY
 
 ### FEATURES
 
+- [node/config: add WithBootstrapPeers option #375](https://github.com/celestiaorg/celestia-node/pull/375) [@Bidon15](https://github.com/Bidon15)
```

Comment:
We don't need changelog for this feature, IMO

Reply:
It's either we don't include it or move it to the improvements section 😓

Reply:
I think we don't need to include every config override feature

Reply:
Ok. Removing the changelog then